examples/private_set_intersection/module/rust/src/tests.rs

```diff
@@ -31,17 +30,21 @@ fn test_set_intersection() {
     let (runtime, entry_channel) = oak_tests::run_single_module_default(MODULE_CONFIG_NAME)
         .expect("Unable to configure runtime with test wasm!");
 
-    let mut req = SubmitSetRequest::new();
-    req.values = RepeatedField::from(vec!["a".to_string(), "b".to_string(), "c".to_string()]);
+    let req = SubmitSetRequest {
+        values: vec!["a".to_string(), "b".to_string(), "c".to_string()].into(),
```

Comment:
I think this may work here:

```suggestion
        values: vec!["a", "b", "c"].into(),
```

Because `String` impls `From<&str>`, and `Vec` forwards the `From` implementation of its contained type.